### ISSUE TITLE
Can't download videos due to too many requests error.

### ISSUE THREAD

**Issue by hollowgalaxy** [activitynet/ActivityNet#51]
CSVs are downloaded from the https://deepmind.com/research/open-source/open-source-datasets/kinetics/
Followed the instructions of ActivityNet/Crawler/Kinetics/README.md to create the environement.
Created `video/` in the same directory as the the download script. 
Made 1 modification to the script to print whenever an error occurs in downloading a file. 
Ran the following command
```python download.py data/kinetics_600_test.csv videos/```

After downloading about 60 videos in, I get the following error for all videos.
```ERROR: Unable to download webpage: HTTP Error 429: Too Many Requests (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.```

**Comment by Pankaj01998** [comment]
Yes I am also getting the same error for all videos after downloading some videos

**Comment by escorciav** [comment]
Thanks for reporting this.

The issue is on the Youtube platform and the `youtube-dl` package. Thus, we don't know how to fix it. Feel free to reference the issue where youtube-dl is tracking the problem.

---

Regarding the datasets that concern to this repo

ActivityNet dataset. You can sign up to download it from [here](https://docs.google.com/forms/d/e/1FAIpQLSeKaFq9ZfcmZ7W0B0PbEhfbTHY41GeEgwsa7WobJgGUhn4DTQ/viewform). You must read the terms and conditions.

For Kinetics dataset, please vote [this petition](https://github.com/activitynet/ActivityNet/issues/28#issuecomment-480499548).

**Comment by Thoudancer** [comment]
@escorciav hello, i filled the form to request the data 3 days ago from [here](https://docs.google.com/forms/d/e/1FAIpQLSeKaFq9ZfcmZ7W0B0PbEhfbTHY41GeEgwsa7WobJgGUhn4DTQ/viewform),but have not received the link mail. How long does it spend to check the mails usually?

**Comment by Frostinassiky** [comment]
@Thoudancer Thanks for your feedback. The server was down for some technique issue. It is running now!

**Comment by jychoi84** [comment]
Hi there, I'm wondering if you have had chance to check my request to download the dataset. I submitted the form last Friday.

**Comment by Frostinassiky** [comment]
> Hi there, I'm wondering if you have had chance to check my request to download the dataset. I submitted the form last Friday.

We met some sharing problem because of the google drive settings. I restart the server and your request will be approved soon!

**Comment by Abrahamon** [comment]
For everyone else, consider installing a newer version of `youtube-dl` through pip, not with apt.
